eng/cmake/global_compile_options.txt

```diff
@@ -1,28 +1,33 @@
 if(MSVC)
-  if(WARNINGS_AS_ERRORS)
-    set(WARNINGS_AS_ERRORS_FLAG "/WX")
-    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /WX")
-  endif()
-
-  add_compile_options(/W4 ${WARNINGS_AS_ERRORS_FLAG} /wd5031 /wd4668 /wd4820 /wd4255 /wd4710 /analyze)
-
   add_compile_options(
-    $<$<CONFIG:>:/MT> #---------|
-    $<$<CONFIG:Debug>:/MTd> #---|-- Statically link the runtime libraries
-    $<$<CONFIG:Release>:/MT> #--|
+      $<$<CONFIG:>:/MT> #---------|
```

Comment:
are the compiler flags are must for sdk? or they are also optional ?

Reply:
these I'm almost positive are a must to build for windows. I'll let @ahsonkhan maybe confirm.